app/models/group.rb

```diff
@@ -0,0 +1,11 @@
+class Group < ActiveRecord::Base
+  GID_CONSTANT = 10000
+
+  validates :name, presence: true
+
+  after_save :set_gid
+
+  def set_gid
```

Comment:
why we generate gid like this?

Reply:
I just copied this from Gate source code, I thought that this will be needed. But if we don't need this in BaritoMarket, then we can delete this.

Reply:
Yes, please make sure that we only write codes necessary for this to function, comply with YAGNI principle